app/frontend/schools/courses/types/students_editor/StudentsEditor__CSVDataError.res

```diff
@@ -31,6 +31,8 @@ let rowNumber = t => t.rowNumber
 
 let errors = t => t.errors
 
+let emails = []
```

Comment:
This doesn't work as intended - unless the page is reloaded after a CSV file is selected, this will cause all loaded emails to be kept in memory, causing uploads of a fixed CSV to complain about all emails previously seen as being dupes.

I'll try to work on a fix for this.

Reply:
552437cb7e0b5bbdf3d04c8b526b785eae100a46 fixes this issue.